dataclients/kubernetes/definitions/routegroupvalidator.go

```diff
@@ -91,6 +93,19 @@ func (rgv *RouteGroupValidator) predicatesValidation(item *RouteGroupItem) error
 	return errorsJoin(errs...)
 }
 
+func (rgv *RouteGroupValidator) validateBackends(item *RouteGroupItem) error {
+	var errs []error
+	for _, backend := range item.Spec.Backends {
+		address, err := url.Parse(backend.Address)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("failed to parse backend address %q: %w", backend.Address, err))
+		} else if address.Path != "" || address.RawQuery != "" {
+			errs = append(errs, fmt.Errorf("backend address %q contains path or query", backend.Address))
+		}
```

Comment:
Should we also check that schema is specified?

Reply:
I don't know if we should do, maybe if we have schema other than (http|https)?  https://github.com/zalando/skipper/blob/master/dataclients/kubernetes/hosts.go#L76-L83

Reply:
Skipper also supports `fastcgi://`. I think checking that its not empty is enough for now, i.e. do not allow `example.com` (should be `https://example.com`)